Packs/Office365MessageTrace/Integrations/Office365MessageTraceEventCollector/Office365MessageTraceEventCollector.yml

```diff
@@ -0,0 +1,138 @@
+  defaultvalue: "1440"
+  display: Processing Delay (minutes)
+  name: processing_delay
+  required: false
+  type: 0
+description: Office 365 Message Trace integration allows you to retrieve email message tracking information from the Office 365 reporting web service. You can search for message traces by date range, sender, recipient, message ID, and status to troubleshoot email delivery issues.
+display: Office 365 Message Trace Event Collector
+name: Office 365 Message Trace Event Collector
+script:
+  script: ''
+  commands:
+  - arguments:
+    - description: Start date for the search in ISO format (YYYY-MM-DDTHH:MM:SSZ). If not provided, defaults to 48 hours ago.
+      name: start_date
+    - description: End date for the search in ISO format (YYYY-MM-DDTHH:MM:SSZ). If not provided, defaults to current time.
+      name: end_date
+    - description: Date range for the search (e.g., "24 hours", "7 days"). Only used if start_date and end_date are not provided.
+      name: date_range
+    - description: Email address of the sender to filter results.
+      name: sender_address
+    - description: Email address of the recipient to filter results.
+      name: recipient_address
+    - description: Specific message trace ID to search for.
+      name: message_trace_id
+    - description: Message status to filter results (e.g., Delivered, Failed, Pending).
+      name: status
+    - description: Number of results to return, will impact paging.
+      name: top
+    description: Retrieve message trace data from Office 365 for troubleshooting email delivery issues.
+    name: office365-mt-get-events
+    outputs:
+    - contextPath: Office365.MessageTrace.MessageTraceId
+      description: Unique identifier for the message trace.
+      type: String
+    - contextPath: Office365.MessageTrace.Organization
+      description: The organization domain that processed the message.
+      type: String
+    - contextPath: Office365.MessageTrace.MessageId
+      description: The Internet MessageID header of the message.
+      type: String
+    - contextPath: Office365.MessageTrace.Received
+      description: Date and time when the message was received.
+      type: Date
+    - contextPath: Office365.MessageTrace.SenderAddress
+      description: SMTP email address of the sender.
+      type: String
+    - contextPath: Office365.MessageTrace.RecipientAddress
+      description: SMTP email address of the recipient.
+      type: String
+    - contextPath: Office365.MessageTrace.Subject
+      description: Subject line of the message.
+      type: String
+    - contextPath: Office365.MessageTrace.Status
+      description: Status of the message (e.g., Delivered, Failed).
+      type: String
+    - contextPath: Office365.MessageTrace.FromIP
+      description: IP address that transmitted the message to Office 365.
+      type: String
+    - contextPath: Office365.MessageTrace.ToIP
+      description: IP address that Office 365 sent the message to.
```

Comment:
The outputs section should be completely removed from event collector, in the python code as well, it's events, not incidents. playbooks will not run on the data.